docs/tutorials/developer-journey/level-4/4.4-nns-governance.md

```diff
@@ -10,7 +10,7 @@ In this tutorial, we'll explore how to use the NNS dapp and how to stake ICP tok
 
 ## What is the NNS?
 
-The Network Nervous System (NNS) is the governing body of the Internet Computer. It is a decentralized autonomous organization that is hosted fully on-chain and is responsible for making protocol-level upgrades to the IC. The goal of the NNS is to allow the IC to be governed in a secure, open, and decentralized way. Therefore, it has complete control over the network. Some examples of upgrades made to the IC through the NNS are:
+The Network Nervous System (NNS) is the governing body of the Internet Computer. It is a decentralized autonomous organization (DAO) that is hosted fully on-chain and is responsible for making protocol-level upgrades to the IC. The goal of the NNS is to allow the IC to be governed in a secure, open, and decentralized way. Therefore, it has complete control over the network. Some examples of upgrades made to the IC through the NNS are:
 
 - Upgrade the protocol and software used by the network's node machines.
 
```

Comment:
A bit further below (I seem to not be able to comment on that line), I propose the change:

1. "When an NNS proposal is submitted, the network participants vote on" > "When an NNS proposal is submitted, the governance participants vote on" (otherwise it might be mistaken for the nodes).
2. "Anyone holding ICP can create a neuron, and are encouraged to do so through reward incentives such as newly minted ICP tokens." > "Anyone holding ICP can create a neuron, and are encouraged to do so through reward incentives." (I think "such as" is a bit misleading as minting tokens is the only way how rewards are paid)
3. "developers can be inspired to turn their application into an SNS as well. " >"developers can be inspired to hand over their application to an SNS DAO as well."